pages/mobile/home.py

```diff
@@ -22,6 +27,29 @@ def __init__(self, base_url, selenium):
     def header_text(self):
         return self.selenium.find_element(*self._header_locator).text
 
+    def is_user_logged_in(self):
```

Comment:
This should be a property, and should return False if the user is not logged in. At the moment it's likely that an exception would be thrown.